sqlalchemy_ibmi/base.py

```diff
@@ -762,6 +763,18 @@ def _get_default_schema_name(self, connection):
             default_schema_name = default_schema_name.strip()
         return self.normalize_name(default_schema_name)
 
+    # Driver version for IBM i Access ODBC Driver is given as
+    # VV.RR.MMSP where VV (major), RR (release), and MM (modificaiton level)
+    # are all two digits, and the SP (service pack level) can be 0, 1, 2, or
```

Comment:
Sorry, I messed this up. The driver version string format is actually VV.RR.SSSF. Where SSS is the service pack and F is a test fix version. We should strip off the testfix version and ignore it.

The current version is `07.01.023` which should map to `[7, 1, 23]`. Right now, the code gives `[7, 1, 2, 3]`.